aten/src/ATen/native/sparse/SparseCsrTensor.cpp

```diff
@@ -322,5 +322,24 @@ const SparseCsrTensor& resize_as_sparse_csr_(
   return self;
 }
 
+SparseCsrTensor clone_sparse_csr(
+    const SparseCsrTensor& self,
+    c10::optional<c10::MemoryFormat> optional_memory_format) {
+  TORCH_CHECK(
+      !optional_memory_format.has_value(),
+      "unsupported memory format option ",
+      optional_memory_format.value());
+  TensorOptions options = self.options();
+  return at::native::_sparse_csr_tensor_unsafe(
+                                               self.crow_indices().clone(optional_memory_format),
```

Comment:
If `optional_memory_format` has a value defined then an error is thrown above with `TORCH_CHECK` so no point to pass it here.

Reply:
Fixed.